share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -830,13 +836,101 @@ e4s-cray-rhel-build:
 e4s-cray-sles-generate:
   extends: [ ".generate-cray-sles", ".e4s-cray-sles" ]
 
-e4s-cray-sles-build:
-  extends: [ ".build", ".e4s-cray-sles" ]
+# e4s-cray-sles-build:
+#   extends: [ ".build", ".e4s-cray-sles" ]
+#   trigger:
+#     include:
+#       - artifact: jobs_scratch_dir/cloud-ci-pipeline.yml
+#         job: e4s-cray-sles-generate
+#     strategy: depend
+#   needs:
+#     - artifacts: True
+#       job: e4s-cray-sles-generate
+
+#######################################
+# E4S - Cray SLES - cce
+#######################################
+.e4s-cray-sles-cce:
```

Comment:
I am worried about the pressure that is currently on the Cray runners and what adding 3 more stacks of similar size will do to them. This change is going to better than double the current load which is already producing wait times ~10x those for other runner types.

https://grafana.spack.io/d/yS3bBliIz/runner-pressure?orgId=1&from=now-2d&to=now

It looks like the current cray pipeline runtime is around 2-3 hours, assuming roughly linear scaling this PR may push that to closer to 6 hours or more between develop pipeline runs.

I think this would be fine to merge, but we should consider finding ways to expand runner capacity for (ie. using the CrayPE containers and AWS resources) or writing other types of stacks that test these compilers in a more scalable environment (ie. ubuntu containers with the target compilers in them). The CrayPE environment is important, but I think testing the compilers has it's own benefit and shouldn't necessarily be lost. The only compiler not available outside of Cray, afiak, is CCE.

Reply:
@kwryankrattiger Todd mentioned this concern to me in a meeting this morning.  

The current plan is to switch to using the official container, which is quite large.  So, at this point (while it's fresh), I am using this PR to get a better handle on how to configure the stacks for the different compilers.

Reply:
@kwryankrattiger I definitely appreciate the information on the expected impact on wait times.